draft-gpew-priv-ppm.md

```diff
@@ -194,22 +194,25 @@ The protocol is executed by a large set of clients and a small set of servers.
 We call the servers the *aggregators*. Each client's input to the protocol is a
 set of measurements (e.g., counts of some user behavior). Given the input set of
 measurements `x_1, ..., x_n` held by `n` users, the goal of a *privacy
-preserving measurement (PPM) protocol* is to compute `y = F(x_1, ..., x_n)` for
-some aggregation function `F` while revealing nothing else about the
-measurements.
+preserving measurement (PPM) protocol* is to compute `y = F(p, x_1, ..., x_n)`
+for some aggregation function `F` and aggregation parameter `p` while revealing
+nothing else about the measurements.
```

Comment:
I think you need to say something more about "p" here. Like, what is it?

And why can't we just bury that in F. I.e., \mathcal{F} is a family of functions parametrized by p and F is a specific instance?

Reply:
Two reasons not to bury this:
1. `F` is implied by the VDAF in use
2. `p` is a parameter chosen by the collector and distributed to the aggregators.

I agree we should say more about what this is. I've added a paragraph to add a bit more color.

By the way, the canonical example of an aggregation parameter is the set of candidate prefixes for each round of heavy hitters.

Reply:
I can live with your text here, but I don't really agree with this rationale. This is introductory text and doesn't need to map 1:1 with the VDAF spec.

Reply:
Yeah I agree. Happy to iterate on it later on.